packages/@okta/vuepress-theme-prose/components/Breadcrumb.vue

```diff
@@ -33,42 +41,53 @@
 
         return true;
       },
-      crumbs() {
-        let crumbs = [];
+      crumbItems() {
+        this.crumbs = [];
+        this.crumbParents = [];
+        var crumbs = this.getCrumbs(this.getNavigation());
```

Comment:
Apologies that the linting isn't setup for this repo yet, but let's prefer `let/const` over `var`